### PR TITLE
CU-2veea54 Create partners component

### PR DESCRIPTION
**Pull request by OlaMikolajczyk**

### CODE REVIEW

**Comment by pmakowski23** [comment]
Task linked: [CU-2veea54 Partners](https://app.clickup.com/t/2veea54)

File: src/components/Partners.astro [hunk 1]
```diff
@@ -0,0 +1,101 @@
+<head>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Inter:wght@600&display=swap" rel="stylesheet">
+</head>
```
Comment (d-wasilewski) [inline src/components/Partners.astro]: Move it to `<head>` in [Layout.astro](https://github.com/tul-ftims-students-council/Delta/blob/master/src/layouts/Layout.astro)

File: src/components/Partners.astro [hunk 2]
```diff
@@ -0,0 +1,101 @@
...
+
+<body>
+    <div class="container">
+        <h2>PARTNERZY STRATEGICZNI</h2>
+        <p>Poznaj firmy bez których wyjazd nie byłby możliwy!</p> 
+        <div class="logos">
+            <img src="/Users/olcia/VSC Delta/Delta/public/assets/partners/NiceGuys.png" height="40" width="249"/>
+            <img src="/Users/olcia/VSC Delta/Delta/public/assets/partners/Deloitte Digital.png" height="80" width="174"/>
+            <img src="/Users/olcia/VSC Delta/Delta/public/assets/partners/Commerzbank.png" height="50" width="381"/>
```
Comment (d-wasilewski) [inline src/components/Partners.astro]: `/assets/partners/file.png` is enough for the file to be resolved

File: src/components/Partners.astro [hunk 3]
```diff
@@ -0,0 +1,101 @@
...
+
+h2, h3 {
+    font-weight: bold;
+    text-align: center;
+    font-family: 'Lemon Milk';
+}
+
+p {
+    font-size: 20px;
+    font-family: 'Inter', sans-serif; /* help */
```
Comment (d-wasilewski) [inline src/components/Partners.astro]: 👍

File: src/components/Partners.astro [hunk 4]
```diff
@@ -0,0 +1,101 @@
...
+    format('woff');
+    font-weight: normal;
+    font-style: normal;
+    }
```
Comment (d-wasilewski) [inline src/components/Partners.astro]: Duplicate of [this](https://github.com/tul-ftims-students-council/Delta/blob/master/src/styles/lemonmilk.css)

**Review by d-wasilewski — CHANGES_REQUESTED** [review]

**Review by d-wasilewski — APPROVED** [review]

**Review by pmakowski23 — APPROVED** [review]

**Pull request merged**